cli/tests/activate.bats

```diff
@@ -3617,3 +3620,110 @@ EOF
   assert_success
   assert_output --regexp "project/.flox/run/.*.project.dev/bin.*default/.flox/run/.*.default.dev/bin"
 }
+
+
+# bats test_tags=fooo
+@test "tcsh: repeat activation in .tcshrc doesn't break aliases" {
+  # We don't need an environment, but we do need wait_for_watchdogs to have a
+  # PROJECT_DIR to look for
+  project_setup_common
+
+  "$FLOX_BIN" init -d default
+  MANIFEST_CONTENTS_DEFAULT="$(cat << "EOF"
+    version = 1
+
+    [profile]
+    tcsh = """
+      alias default_alias echo "Hello default!";
+    """
+EOF
+  )"
+  echo "$MANIFEST_CONTENTS_DEFAULT" | "$FLOX_BIN" edit -d default -f -
+
+  "$FLOX_BIN" init -d project
+  MANIFEST_CONTENTS_PROJECT="$(cat << "EOF"
+    version = 1
+
+    [profile]
+    tcsh = """
+      alias project_alias echo "Hello project!";
+    """
+EOF
+  )"
+  echo "$MANIFEST_CONTENTS_PROJECT" | "$FLOX_BIN" edit -d project -f -
+
+  echo "eval \`$FLOX_BIN activate -d '$PROJECT_DIR/default'\`" > "$HOME/.tcshrc.extra"
+
+  # It would be better use tcsh -i to source .tcshrc,
+  # but that causes the tests to background because tcsh -i tries to open
+  # /dev/tty.
+  # Instead `flox activate -d default` manually to simulate sourcing
+  # .tcshrc
+
+  TCSH="$(which tcsh)"
+  EXPECT="$(which expect)"
+  run tcsh <(cat <<'EOF'
+    if (alias default_alias) then
+    else
+      echo "default_alias not found"
+      exit 1
+    endif
+    # We can't double check the alias has been loaded because bash isn't
+    # interactive and discards it
+    # setenv FLOX_SHELL "$TCSH"
+    # setenv NO_COLOR 1
+    # "$EXPECT" "$TESTS_DIR/activate/activate-command.exp" "$PROJECT_DIR/project" "which project_alias && which default_alias"
+EOF
+)
+  assert_success
+  assert_output --partial "project_alias is aliased to \`echo Hello project!'"
+  assert_output --partial "default_alias is aliased to \`echo Hello default!'"
+}
```

Comment:
currently this second to last test is still failing for me with 

```
   -- command failed --
   status : 1
   output (6 lines):
     Sourcing .tcshrc
     Setting PATH from .tcshrc
     ["path":"/private/tmp/nix-shell.8XZ85u/nix-shell.T20ChN/bats-run-tbaacv/test/1/project-1/default/.flox"]: Command not found.
     ["pointer":"name":"default"]: Command not found.
     ["pointer":"version":1]: Command not found.
     if: Expression Syntax.
   --
```

that's likely the 

```
    if (alias default_alias) then
    else
      echo "default_alias not found"
      exit 1
    endif
```

block or something in the generated script, `if: Expression Syntax.` is a wonderfully clear error in either case.